src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -1060,10 +1072,10 @@ export default class DefaultDeviceController
       // Some browsers will not play audio out the MediaStreamDestination
       // unless there is actually audio to play, so we add a small amount of
       // noise here to ensure that audio is played out.
-      source.buffer.getChannelData(0)[0] = 0.0003;
-      source.loop = true;
-      source.connect(outputNode);
-      source.start();
+      this.audioBufferSourceNode.buffer.getChannelData(0)[0] = 0.0003;
```

Comment:
>Should this be a class variable or instance variable? Technically you can have multiple device controllers that call this right?

Yes, as we discussed earlier in the standup, it's better to be a private property. However the `createEmptyAudioDevice` is a static method, it can not access the private property. If we update `createEmptyAudioDevice` to private method, it's a breaking change.

>Also can't we just set sourceNode then assign it at the end before return to audioBufferSourceNode. Not sure why we need to replace everywhere like this.

Good point. Updated!

Reply:
You can just return the node itself instead of the media stream right? Then you can set it to an instance variable.
```
static synthesizeAudioDevice(toneHz: number): audioBufferSourceNode {
```

I think you can create a new method with same name but return node instead.

Reply:
>You can just return the node itself instead of the media stream right?

I don't think so. When creating an empty device, the stream it returns is from `MediaStreamAudioDestinationNode`.
```
AudioBufferSourceNode ---connect--> MediaStreamAudioDestinationNode --> stream (returned).
```

If we return `audioBufferSourceNode` directly, then we need create the MediaStreamAudioDestinationNode and connect AudioBufferSourceNode to MediaStreamAudioDestinationNode and get/return the stream. Feel like it kinda break the change.